scss/templates/user.scss

```diff
@@ -289,38 +308,45 @@
     margin-bottom: 1.5rem;
   }
 
-  .top-sub-section ul li {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    min-height: 3.25rem;
-    margin-bottom: 0.8rem;
+  .top-sub-section ul {
+    max-width: 100%;
 
-    a {
-      order: 1;
-      max-width: 85%;
-      font-style: normal;
-      font-weight: bold;
-      font-size: 0.875rem;
-      line-height: 1.0625rem;
-      color: $gray-100;
-    }
+    li {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      min-height: 3.25rem;
+      margin-bottom: 0.8rem;
 
-    .topic-info {
-      order: 2;
-    }
+      a {
+        order: 1;
+        max-width: 85%;
```

Comment:
🤔 You know what I am going to say hahaha... but it is you call, as I don't know if there is some special case or whatsoever, nevertheless, yesterday we saw the often this dimensions are not needed.

Reply:
Yeah, this is weird. I'll take a look on that haha

Reply:
I just removed it, it was not required